zipkin_v2_span.go

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jaeger
+
+import (
+	"net"
+
+	"github.com/opentracing/opentracing-go/ext"
+
+	zipkinModel "github.com/openzipkin/zipkin-go/model"
+	"github.com/uber/jaeger-client-go/internal/spanlog"
+	"github.com/uber/jaeger-client-go/utils"
+)
+
+// BuildZipkinV2Span converts a Jaeger span to a Zipkin v2 span
+func BuildZipkinV2Span(span *Span) *zipkinModel.SpanModel {
```

Comment:
is there a way to stash these files under `zipkin/` package? We don't want to increase the footprint of the client for users who only use it with non-zipkin parts

Reply:
@black-adder I think this would also allow the zipkin-go lib to be only pulled in if projects were using code from the zipkin subpackage (at least this is my understanding of how glide/dep work).